public/cli/lib/commands/ffmpeg.js

```diff
@@ -128,19 +130,20 @@ function checkAudioInput(done) {
             let mp3Files = files.filter((f) => f.indexOf(".mp3") > -1),
                 wavFiles = files.filter((f) => f.indexOf(".wav") > -1),
                 audioFiles = [];
-
             // If this folder contains wave and mp3 files, then throw error
             if (mp3Files.length && wavFiles.length) {
                 return done(new Error("Conflicting audio types"));
             } else if (mp3Files.length) {
                 // can use glob format with .mp3 files
-                audioFiles = mp3Files;
+                audioFiles = mp3Files.filter((f)=>{ return Options.skipAudioFiles.indexOf(f) == -1; });
```

Comment:
Non-blocking: This can be simplified to `audioFiles = mp3Files.filter(f => Options.skipAudioFiles.indexOf(f) == -1);`. An implied return with no code block. Here and elsewhere.

Reply:
For changes like this, we should probably come up with a set of eslint rules for the project, and have some standard tool to auto correct these things.  We have recently moved to eslint + prettier for our projects.  Takes a lot of the decision making out of the programmers hands as it pertains to code formatting which should result in much more uniform code.  

On the flip side, it also means some code blocks look a bit more "janky" due to the prettier rules that we can't change.

Reply:
so for the moment, let's leave these as they are since they are valid javascript and wait for a common code linting solution.

Reply:
Sure, this was non-blocking anyway. We use prettier also so happy for you to add it to the project or make an issue for it.